internal/auth/auth.go

```diff
@@ -0,0 +1,26 @@
+package auth
+
+import (
+	"github.com/dgrijalva/jwt-go"
+	"github.com/labstack/gommon/random"
+	"os"
+)
+
+func CreateNewToken() (string, error) {
+	// set claims
+	token := jwt.NewWithClaims(jwt.SigningMethodHS256, jwt.MapClaims{
+		"authorized": true,
+	})
+	return token.SignedString([]byte(getSecret()))
+}
+
+func getSecret() string {
+	secret := os.Getenv("AUTH_SECRET")
```

Comment:
secret should be passed and param, since we should accept both secret as cli param or env variable.
Return error if missing as generating new secret on restart would invalidate all existing tokens.